bugzooka/integrations/gemini_client.py

```diff
@@ -12,6 +12,7 @@
     INFERENCE_MAX_TOOL_ITERATIONS,
 )
 from bugzooka.integrations.inference import InferenceAPIUnavailableError
+from bugzooka.analysis.prompts import JIRA_TOOL_PROMPT
```

Comment:
Most of the changes here are related to formatting, can we avoid them in this PR please and only keep the real code changes? Those optimizations can be taken care in other follow ups if requried.